package ExceptionsPackage;

public class CompletedTaskException extends Exception{
}
